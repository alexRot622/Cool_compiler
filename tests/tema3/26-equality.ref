Loaded: trap.handler.nogc
The following symbols are undefined:
main

3 OK
a OK
a1 OK
a2 failed
b OK
b1 OK
b2 failed
x OK
self OK
comp failed
copy failed
COOL program successfully executed
