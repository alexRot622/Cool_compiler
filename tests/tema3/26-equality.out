Loaded: trap.handler.nogc
The following symbols are undefined:
main

3 OK
a OK
a1 OK
a2 OK
b OK
b1 OK
b2 OK
x OK
self OK
comp OK
copy OK
COOL program successfully executed
